pcdscalc/constants.py

```diff
@@ -0,0 +1,345 @@
+    "Ge": (5.658, 5.658, 5.658, 90, 90, 90),
+    "As": (4.1018, 4.1018, 4.1018, 54.554, 54.554, 54.554),
+    "Kr": (5.64, 5.64, 5.64, 90, 90, 90),
+    "Rb": (5.585, 5.585, 5.585, 90, 90, 90),
+    "Sr": (6.08, 6.08, 6.08, 90, 90, 90),
+    "Y": (3.65, 3.65, 5.73, 90, 90, 120),
+    "Zr": (3.23, 3.23, 5.15, 90, 90, 120),
+    "Nb": (3.3, 3.3, 3.3, 90, 90, 90),
+    "Mo": (3.15, 3.15, 3.15, 90, 90, 90),
+    "Tc": (2.74, 2.74, 4.4, 90, 90, 120),
+    "Ru": (2.71, 2.71, 4.28, 90, 90, 120),
+    "Rh": (3.8, 3.8, 3.8, 90, 90, 90),
+    "Pd": (3.89, 3.89, 3.89, 90, 90, 90),
+    "Ag": (4.09, 4.09, 4.09, 90, 90, 90),
+    "Cd": (2.98, 2.98, 5.62, 90, 90, 120),
+    "In": (3.25, 3.25, 4.95, 90, 90, 90),
+    "Sn": (6.49, 6.49, 6.49, 90, 90, 90),
+    "Sb": (4.4898, 4.4898, 4.4898, 57.233, 57.233, 57.233),
+    "Xe": (6.13, 6.13, 6.13, 90, 90, 90),
+    "Cs": (6.045, 6.045, 6.045, 90, 90, 90),
+    "Ba": (5.02, 5.02, 5.02, 90, 90, 90),
+    "Ce": (5.16, 5.16, 5.16, 90, 90, 90),
+    "Eu": (4.58, 4.58, 4.58, 90, 90, 90),
+    "Gd": (3.63, 3.63, 5.78, 90, 90, 120),
+    "Tb": (3.6, 3.6, 5.7, 90, 90, 120),
+    "Dy": (3.59, 3.59, 5.65, 90, 90, 120),
+    "Ho": (3.58, 3.58, 5.62, 90, 90, 120),
+    "Er": (3.56, 3.56, 5.59, 90, 90, 120),
+    "Tm": (3.54, 3.54, 5.56, 90, 90, 120),
+    "Yb": (5.45, 5.45, 5.45, 90, 90, 90),
+    "Lu": (3.5, 3.5, 5.55, 90, 90, 120),
+    "Hf": (3.19, 3.19, 5.05, 90, 90, 120),
+    "Ta": (3.3, 3.3, 3.3, 90, 90, 90),
+    "W": (3.16, 3.16, 3.16, 90, 90, 90),
+    "Re": (2.76, 2.76, 4.46, 90, 90, 120),
+    "Os": (2.74, 2.74, 4.32, 90, 90, 120),
+    "Ir": (3.84, 3.84, 3.84, 90, 90, 90),
+    "Pt": (3.92, 3.92, 3.92, 90, 90, 90),
+    "Au": (4.08, 4.08, 4.08, 90, 90, 90),
+    "Tl": (3.46, 3.46, 5.52, 90, 90, 120),
+    "Pb": (4.95, 4.95, 4.95, 90, 90, 90),
+    "Bi": (4.7236, 4.7236, 4.7236, 57.35, 57.35, 57.35),
+    "Po": (3.34, 3.34, 3.34, 90, 90, 90),
+    "Ac": (5.31, 5.31, 5.31, 90, 90, 90),
+    "Th": (5.08, 5.08, 5.08, 90, 90, 90),
+    "Pa": (3.92, 3.92, 3.24, 90, 90, 90),
+    "ZnSe": (5.6676, 5.6676, 5.6676, 90, 90, 90),
+    "ZnTe": (6.101, 6.101, 6.101, 90, 90, 90),
+    "CdS": (5.832, 5.832, 5.832, 90, 90, 90),
+    "CdSe": (6.05, 6.05, 6.05, 90, 90, 90),
+    "CdTe": (6.477, 6.477, 6.477, 90, 90, 90),
+    "BN": (3.615, 3.615, 3.615, 90, 90, 90),
+    "GaSb": (6.0954, 6.0954, 6.0954, 90, 90, 90),
+    "GaAs": (5.65315, 5.65315, 5.65315, 90, 90, 90),
+    "GaMnAs": (5.65, 5.65, 5.65, 90, 90, 90),
+    "GaP": (5.4505, 5.4505, 5.4505, 90, 90, 90),
+    "InP": (5.86875, 5.86875, 5.86875, 90, 90, 90),
+    "InAs": (6.05838, 6.05838, 6.05838, 90, 90, 90),
+    "InSb": (6.47877, 6.47877, 6.47877, 90, 90, 90),
+    "LaMnO3": (5.531, 5.602, 7.742, 90, 90, 90),
+    "LaAlO3": (5.377, 5.377, 5.377, 60.13, 60.13, 60.13),
+    "La0.7Sr0.3MnO3": (5.4738, 5.4738, 5.4738, 60.45, 60.45, 60.45),
+    "Gd3Ga5O12": (12.383, 12.383, 12.383, 90, 90, 90),
+}
+
+# define units and constants
+units = {
```

Comment:
While this is fine for now, we may consider relying on `pint` as we do in pcdsdevices:

https://github.com/pcdshub/pcdsdevices/blob/5ee91018658423b297e17a242f72b09bfcefc87e/pcdsdevices/utils.py#L93-L122

Probably should open an issue for this.

Reply:
Opened an issue: https://github.com/pcdshub/pcdscalc/issues/10 - thank you for the suggestion